test/testfiles/openapi-schema.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$id": "openapi-schema",
+  "properties": {
+    "description": {
```

Comment:
Remove?

Reply:
Replaced it with a custom schema file to test possibility to use other than default schema to parse openapi files.